src/main/python/systemds/operator/nodes/matrix.py

```diff
@@ -522,5 +522,12 @@ def replace(self, pattern: VALID_INPUT_TYPES, replacement: VALID_INPUT_TYPES) ->
         """
         return Matrix(self.sds_context, "replace", named_input_nodes={"target": self, "pattern": pattern, "replacement": replacement})
 
+    def cumsum(self) -> 'Matrix':
+        """ Column prefix-sum. (For row-prefix sum, use X.t().cumsum().t())
```

Comment:
interesting, do we not support cumulative sum on rows?

Reply:
jep, afaik. It says so in the docu (even Arnab told me yesterday not trust it ^^) and java code suggests it too

Reply:
@mboehm7 

I think we should probably make a task for this, if it really is not supported.
It seems odd to to two transposes to do row cumulative operations.

Reply:
well, its fine to add a task even through the focus was always R compliance and are defines it as column-wise cumulative aggregates (unary operation without additional parameters). The same also applies to rev(). Recently some of the operations like roll() are more inspired by numpy and there you have indeed usually axis parameters.

Reply:
Also when committing this task here lets call it [SYSTEMDS-3744-48] ... and in the future let's create JIRA tasks for groups of operations that belong together. Thanks.

Reply:
i have made a task for it and marked it as a student project, unrelated to this PR, such that we do not forget it.